container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/logs_aggregator_functions/implementations/vector/consts.go

```diff
@@ -29,7 +31,9 @@ const (
 	// To construct the filepath, we utilize vectors template syntax that allows us to reference fields in log events
 	// https://vector.dev/docs/reference/configuration/template-syntax/
 	baseLogsFilepath = "\"" + logsStorageDirpath + "%%G/%%V/"
-
-	uuidLogsFilepath = baseLogsFilepath + "{{ enclave_uuid }}/{{ service_uuid }}.json\""
 	////////////////////////--FINISH--VECTOR CONFIGURATION SECTION--/////////////////////////////
 )
+
+var (
+	uuidLogsFilepath = baseLogsFilepath + fmt.Sprintf("{{ %v }}/{{ %v }}.json\"", docker_label_key.LogsEnclaveUUIDDockerLabelKey.GetString(), docker_label_key.LogsServiceUUIDDockerLabelKey.GetString())
+)
```

Comment:
updated this to be created dynamically so that uuid logs file path is always based on what the logs labels